src/lib/accessibility/interfaces.ts

```diff
@@ -178,4 +178,4 @@ export type ActionsKeyHandler = {
 export type KeyboardHandler = (event: KeyboardEvent) => void
 export type EventHandler = (event: Event) => void
 
-export type Accessibility = IAccessibilityDefinition | ((props: any) => IAccessibilityDefinition)
+export type Accessibility = ((props: any) => IAccessibilityDefinition)
```

Comment:
'Accessibility' is a very generic word, should we consider to change it to something more meaningful

Reply:
yes, quite generic, any ideas of a name?
Currently, we already have ```IAccessibilityDefinition```, ```IAccessibilityBehavior```

Reply:
what do you think about IAcessibilityProvider?

Reply:
it's a type, so it will be AccessibilityProvider. Can be like that, we need more opinions and ideas :) @jurokapsiar what do you think on naming?

Reply:
I personally think Accessibility is ok, if there's concern, than it should probably be AccessibilityBehavior

Reply:
yeah, for me both are ok. 
But guys, let's better merge this PR if the changes are ok for you. 
And decide on naming other time, maybe we'll get inspired and find the perfect name :)